### DISCUSSION TITLE
The background changes itself when I change the virtual desktop

### DISCUSSION THREAD

**Discussion by Roxale77** [valinet/ExplorerPatcher#3238] [Q&A]
Hello, yesterday, when I was trying to update the program, it seems that the antivirus caused an error whereby, when I changed the virtual desktop, the background that was in presentation was changed only to an image.

To try to solve the problem I uninstalled the program and restarted the computer, but when I tried to see if the problem had been solved it was still there.

Therefore I would like to know if there is any way to solve this.

Sorry for the english, i used google translate.

Explorer patcher version: 22621.3527.65.1

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

**Comment by Roxale77** [comment]
Windows version: 23H2
Build: 22631.3527
I have Winstep Nexus (i don't know if is a UI)
![image](https://github.com/valinet/ExplorerPatcher/assets/169074549/76669f97-f604-418d-a70d-570ce3412c5c)

**Comment by pyrates999** [comment]
Please show screen shot of what you are seeing for your desktop background.

**Comment by Amrsatrio** [comment]
It is a feature in Windows where you can change the desktop wallpaper of a Virtual Desktop, accessed by right clicking the Virtual Desktop tile in the Task View.

**Comment by Roxale77** [comment]
https://github.com/valinet/ExplorerPatcher/assets/169074549/e992b89e-3f97-4821-a292-188d9448776f

In this video you can see the problem that i have

**Comment by pyrates999** [comment]
This looks to be a bug in windows itself since you have uninstalled EP already and it is still happening.

**Comment by Roxale77** [comment]
Oh, ok.
Thanks for your time.
